Dockerfile

```diff
@@ -1,4 +1,6 @@
-FROM node:10.15.1
+FROM node:10.16.0
+ARG codeServerVersion=docker
```

Comment:
We used to pull the binary from the docker image that gets built. Is that still done? If so the distributed binary would have `codeServerVersion=docker` as well.

Reply:
For the CI right? The CI doesn't use this Dockerfile at all; it builds using `codercom/nbin-centos` or `codercom/nbin-alpine` and the binary builds with the version specified in the Travis config.

For the `codercom/code-server` image we've been using this Dockerfile and building it manually then pushing it and it would just always say "development" for the version. Now we can specify the version when building it (`docker` is just a default value).

The ideal would probably be to have Travis push `codercom/code-server` so we can just use the version specified by Travis and not have to manage it manually. I'm not sure exactly how we want to orchestrate that so for this PR I just left the overarching build process the same.

Reply:
Ah right. Makes sense